db/migrate/20161115171029_add_encrypted_pii_recovery_code.rb

```diff
@@ -0,0 +1,5 @@
+class AddEncryptedPiiRecoveryCode < ActiveRecord::Migration
+  def change
+    add_column :profiles, :encrypted_pii_recovery, :text
```

Comment:
the column has encrypted pii for the purposes of recovery. it's not storing the recovery code.